packages/create-discord-bot/template/JavaScript/src/index.js

```diff
@@ -9,7 +9,7 @@ const eventFiles = await readdir(eventsPath).then((files) => files.filter((file)
 
 for (const file of eventFiles) {
 	const event = (await import(join(eventsPath, file))).default;
```

Comment:
Also i am not too sure about that [message](https://discord.com/channels/222078108977594368/222197033908436994/1104848759550124043)

Reply:
Review comments are not where you raise unrelated issues... there is an issue tracker for this.

Reply:
Yeah i know